src/main/resources/db/changelog/changeLog_v1.json

```diff
@@ -127,6 +127,93 @@
           }
         ]
       }
+    },
+
+    {
+      "changeSet": {
+        "id": "3",
+        "author": "Salam",
+        "changes": [
+          {
+            "createTable": {
+              "tableName": "task",
+              "columns": [
+                {
+                  "column": {
+                    "name": "id",
+                    "type": "bigint",
+                    "autoIncrement": true,
+                    "constraints": {
+                      "primaryKey": true
+                    }
+                  }
+                },
+                {
+                  "column": {
+                    "name": "title",
+                    "type": "varchar(255)",
+                    "constraints": {
+                      "nullable": false,
+                      "unique": true
+                    }
+                  }
+                },
+                {
+                  "column": {
+                    "name": "description",
+                    "type": "varchar(255)"
+                  }
+                },
+                {
+                  "column": {
+                    "name": "due_date",
+                    "type": "timestamp with time zone"
+                  }
+                },
+                {
+                  "column": {
```

Comment:
instead of completed attribute create an attribute named status

Reply:
Done in #6